src/pfe/file-watcher/server/src/utils/locales/en/translation.json

```diff
@@ -75,6 +75,7 @@
   },
   "projectUtil": {
     "projectInfoError": "could not retrieve project info",
+    "serverLessApp": "server less application detected. app status set to stopped",
```

Comment:
Can we update the key to something similar to the variable? eg. "pingDisabled": "The port is set to -1, pinging is disabled and that app status is set to Stopped."